cpp/util/libevent_wrapper.cc

```diff
@@ -272,7 +272,10 @@ event* Base::EventNew(evutil_socket_t& sock, short events,
 
 
 evhttp* Base::HttpNew() const {
-  return CHECK_NOTNULL(evhttp_new(base_.get()));
+  evhttp* vhtp=CHECK_NOTNULL(evhttp_new(base_.get()));
```

Comment:
Not sure that `vhtp` is that great of a name? I'm thinking something simple, like `http` (since the method is called `HttpNew`), ought to do it?

Also, mind the use of whitespace: there should be a single space on both sides of a `=`, and a single space after a `,` (below).

I personally prefer "constructor syntax", such as at line 380 of this file.